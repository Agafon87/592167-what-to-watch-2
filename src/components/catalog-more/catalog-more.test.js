import React from "react";
import renderer from "react-test-renderer";
import CatalogMore from "./catalog-more.jsx";

it(`CatalogMore correctly renderer after test`, () => {
  const tree = renderer.create(<CatalogMore
    films={[]}
    filmsCount={1}
    onCatalogMoreClick={jest.fn}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
