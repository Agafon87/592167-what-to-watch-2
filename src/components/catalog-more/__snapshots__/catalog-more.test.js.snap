// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatalogMore correctly renderer after test 1`] = `
<div
  className="catalog__more invisible-block"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
